### PR TITLE
:tada: completed alert and avatar components

### PR DESCRIPTION
**Pull request by rahulk31**
Please ignore gist.css file 😃

### CODE REVIEW

**Comment by Sahil25061999** [comment]
looks good to me , can be merged. Beautiful UI by the way.

**Comment by rahulk31** [comment]
> looks good to me , can be merged. Beautiful UI by the way.

Thank you @Sahil25061999 for the review. 😃

File: documentation.html [hunk 1]
```diff
@@ -59,7 +59,194 @@
...
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-xl">
+                <img src="./assets/avatar/marathi-woman.svg" alt="" />
+              </div>
+            </div>
+
+            <div class="avatar-wrapper">
+              <div class="avatar avatar-xxl">
+                <img src="./assets/avatar/rajasthani-male.svg" alt="" />
```
Comment (Naman-Saxena1) [inline documentation.html]: alts are empty for images.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Overall looks fine 👍

**Comment by swstica** [comment]
Looks good to me, although the "get started" font in navbar is a bit small IMO.

File: documentation.html (hunk 1 above)
Comment (rahulk31) [reply documentation.html]: Thanks Naman for pointing these, will provide some text asap 😄

**Comment by rahulk31** [comment]
> Looks good to me, although the "get started" font in navbar is a bit small IMO.

Thanks Swastika for the review, I will get it fixed in next PR as I am planning to apply the button component class on that. 😃

**Comment by gauravsinhaweb** [comment]
get started text size looks a bit odd. Good to merge after changes. Nice work!

**Comment by rahulk31** [comment]
> get started text size looks a bit odd. Good to merge after changes. Nice work!

Thanks Gaurav for the review, I will implement button component and add it to the Get started text. 😃

File: documentation.html (hunk 1 above)
Comment (rahulk31) [reply documentation.html]: Have added alt texts:) Merging PR.

**Pull request merged**